test/NuGet.Core.FuncTests/NuGet.Commands.FuncTest/RestoreCommandTests.cs

```diff
@@ -4697,13 +4698,13 @@ await SimpleTestPackageUtility.CreateFolderFeedV3Async(
                 }
             }
           }
-        }";
+        }".Replace("FRAMEWORK", framework);
 
             // Setup project
             var projectSpec = ProjectTestHelpers.GetPackageSpecWithProjectNameAndSpec("Project1", pathContext.SolutionRoot, rootProject);
             projectSpec.RestoreMetadata.SdkAnalysisLevel = !string.IsNullOrEmpty(sdkAnalysisLevel) ? NuGetVersion.Parse(sdkAnalysisLevel) : null;
             projectSpec.RestoreMetadata.UsingMicrosoftNETSdk = usingMicrosoftNETSdk;
-            var projectSpec2 = ProjectTestHelpers.GetPackageSpec("Project2", framework: "net472");
+            var projectSpec2 = ProjectTestHelpers.GetPackageSpec("Project2", framework: framework);
```

Comment:
Is framework the second parameter? If so, we don't need the to name the parameter since the variable passed is already self-explanatory.

```suggestion
            var projectSpec2 = ProjectTestHelpers.GetPackageSpec("Project2", framework);
```

Reply:
you're right. 
Pushed an updated.